temporal/assets/monitors/FrontendLatency.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "Temporal frontend latency is elevated",
+	"type": "query alert",
+	"query": "avg(last_5m):sum:temporal.server.service.latency.sum{!operation:poll*,*,service_name:frontend}.as_rate() / sum:temporal.server.service.latency.count{!operation:poll*,*,service_name:frontend}.as_rate() > 1000",
```

Comment:
What's the purpose of the `*` (here: poll*,*****,service_name)?

Reply:
Looks like a mistake, thanks for finding that!